libs/brackets-server/embedded-ext/file-extension/main.js

```diff
@@ -292,39 +291,57 @@ define(function (require, exports, module) {
         }
     }
 
+    function handleDownloadFile() {
+        var selected = ProjectManager.getSelectedItem();
+
+        if (selected && selected.isFile) {
+            var projectId = PreferencesManager.getViewState("projectId");
```

Comment:
var -> const ?

Reply:
did you run grunt htmllint ?
IMO, need to consider this as new file.

Reply:
Done.